SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -85,7 +85,7 @@ void SpatialVirtualWorkerTranslator::ApplyMappingFromSchema(Schema_Object* Objec
 {
 	if (!IsValidMapping(Object))
 	{
-		UE_LOG(LogSpatialVirtualWorkerTranslator, Log,
+		UE_LOG(LogSpatialVirtualWorkerTranslator, Verbose,
 			   TEXT("Received invalid mapping, likely due to PiE restart, will wait for a valid version."));
```

Comment:
I don't think is the most likely cause of this issue anymore, it much more often happens due to the user launching a mismatched number of workers. Also, in terms of the spam that comes from servers, I think this is the one line we should leave in. However, it should read something more like "Waiting for virtual worker translation mapping to be ready" AND would ideally feature something inciteful about the current mapping from IsValidMapping, e.g. there are only 3 workers in the mapping when there should be 4. This would make debugging a LOT easier.

Reply:
IsValidMapping does not check that we have the expected number of worker, it only checks we received a mapping that contains the local worker. I get why it would be useful to report the number of workers we found, but I'll have to find another place to put the log.

Reply:
> it only checks we received a mapping that contains the local worker.

Sounds good, can we change the log to say something like: "Waiting for this server to be assigned a virtual worker from the load balancing strategy"